styles/_mandara.scss

```
@@ -243,6 +243,164 @@ body:has(.mandara-editor) {
+    font-size: var(--font-size-md);
+    resize: vertical;
+
+    &:focus {
+      outline: none;
+      border-color: var(--color-accent);
+    }
+  }
+
+  .import-actions {
+    @include flex(start, center);
+    gap: var(--spacing-sm);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)